The Native build image are using java8 (should java 11) to compile the project

Adding maven to the mandrel image also installs java8 into the image and sets the default java version to 1.8. When trying to compile a Quarkus project using `<maven.compiler.target>11</maven.compiler.target>` will fail to build with the following error message.

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project sample-app: Fatal error compiling: invalid target release: 11 -> [Help 1]
```

--- a/apps/quarkus-sample-app/pom.xml
+++ b/apps/quarkus-sample-app/pom.xml
@@ -8,8 +8,8 @@
   <properties>
     <compiler-plugin.version>3.8.1</compiler-plugin.version>
     <maven.compiler.parameters>true</maven.compiler.parameters>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <quarkus-plugin.version>1.8.1.Final</quarkus-plugin.version>
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>

--- a/apps/quarkus-sample-app/src/main/resources/application.properties
+++ b/apps/quarkus-sample-app/src/main/resources/application.properties
@@ -1,2 +1,5 @@
 # Quarkus Configuration file
 # key = value
+
+# Limits the build memory during the native build to 1.5G, increase or disable if you prefer a fast build without concerns about memory utilization.
+quarkus.native.additional-build-args=-J-Xmx1536m 

--- a/buildpacks/quarkus/bin/build
+++ b/buildpacks/quarkus/bin/build
@@ -2,7 +2,7 @@
 
 set -eo pipefail
 
-echo "---> Quarkus JVM Buildpack"
+echo "---> Quarkus Buildpack"
 
 # GET ARGS
 layersdir=$1
@@ -14,6 +14,8 @@ mkdir -p "$m2layer"
 mkdir -p $HOME/.m2
 ln -s "$m2layer" $HOME/.m2/repository
 echo -e 'cache = true' > "$m2layer.toml"
+
+export JAVA_HOME=/usr/lib/jvm/java-11-openjdk
 
 # BUILD THE APPLICATION
 if [[ -f pom.xml ]]; then

--- a/stacks/jvm/build/Dockerfile
+++ b/stacks/jvm/build/Dockerfile
@@ -6,9 +6,6 @@ FROM ${base_image}
 #ARG cnb_gid=185
 ARG stack_id
 
-USER root
-RUN dnf --setopt=tsflags=nodocs install -y maven
-
 # Set required CNB information
 
 ENV CNB_STACK_ID=${stack_id}

